core/src/main/java/org/apache/brooklyn/core/mgmt/ha/HighAvailabilityManagerImpl.java

```diff
@@ -999,6 +1000,15 @@ public ManagementPlaneSyncRecord loadManagementPlaneSyncRecord(boolean useLocalK
         return record; 
     }
 
+    private boolean updateLastManagementPlaneSyncRecordWithLocalKnowledge() {
+        if (lastSyncRecord!=null) {
+            lastSyncRecord = updateManagementPlaneSyncRecordWithLocalKnowledge(lastSyncRecord);
+            return true;
+        } else {
+            return false;
```

Comment:
Any plans for the return value? Javadoc would help.

Reply:
We don't need to return value for the particular scenario this change addresses (as it is then retrieved via a separate API call). In any case it can be retrieved by calling getLastManagementPlaneSyncRecord()  so I think it is not necessary, happy to be proven wrong of course